packages/arb-token-bridge-ui/src/token-bridge-sdk/OftV2TransferStarter.ts

```diff
@@ -280,7 +279,7 @@ export class OftV2TransferStarter extends BridgeTransferStarter {
     await this.validateOftTransfer()
 
     const config = await prepareTransferConfig({
-      signer,
+      from: await signer.getAddress(),
```

Comment:
why doesn't this one use `await getAddressFromSigner(signer)`?
or maybe the others can be changed to this? just for consistency